docker-compose.db.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Compose file for a "persistent" database whose data will be kept in db_volume.
+version: "3.8"
+services:
+  db:
+    image: datarttu/sujuikodb:latest
+    build: .
+    restart: unless-stopped
+    env_file: .env
+    ports:
+      - 127.0.0.1:${POSTGRES_LOCAL_PORT}:5432
+    volumes:
+      - type: bind
+        source: ./db/db_test
+        target: /docker-entrypoint-initdb.d/
+        read_only: true
+      - db_volume:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "${POSTGRES_USER}"]
+      interval: 30s
+      start_period: 5s
+      timeout: 20s
+      retries: 3
+  dataimporter:
```

Comment:
As suggested in a previous comment, `dataimporter` could be separated into a script.